.github/workflows/main.yml

```diff
@@ -200,10 +204,12 @@ jobs:
         | text    | ${{ needs.build-firmware.outputs.text_size }}B | ${{ steps.output-sizes-diff.outputs.text_diff }}B |
         | data    | ${{ needs.build-firmware.outputs.data_size }}B | ${{ steps.output-sizes-diff.outputs.data_diff }}B |
         | bss     | ${{ needs.build-firmware.outputs.bss_size }}B  | ${{ steps.output-sizes-diff.outputs.bss_diff }}B  |
+
+        [Run on InfiniEmu](https://infiniemu.pipe01.net/?firmware=https://corsproxy.io/?https://nightly.link/${{ github.repository }}/actions/artifacts/${{ needs.build-firmware.outputs.firmware_artifact }}.zip&resources=https://corsproxy.io/?https://nightly.link/${{ github.repository }}/actions/artifacts/${{ needs.build-firmware.outputs.resources_artifact }}.zip)
```

Comment:
Would it make more sense to say "Run in InfiniEmu"?

Reply:
Probably, I'm never sure which one to use haha